src/internal/account.ts

```diff
@@ -795,3 +803,72 @@ export async function isAccountExist(args: { aptosConfig: AptosConfig; authKey:
     throw new Error(`Error while looking for an account info ${accountAddress.toString()}`);
   }
 }
+
+export async function verifySecp256k1Account(args: {
+  aptosConfig: AptosConfig;
+  message: HexInput;
+  signature: HexInput | Secp256k1Signature | AnySignature;
+  recoveryBit?: number;
+  accountAddress: AccountAddressInput;
+}): Promise<AnyPublicKey> {
+  const { aptosConfig, message, recoveryBit, accountAddress } = args;
+  let signature: HexInput | Secp256k1Signature;
+  if (args.signature instanceof AnySignature) {
```

Comment:
`instanceOf` is known to be causing a lot of problem, specifically when using the TS sdk from different bundles
See: https://github.com/aptos-labs/aptos-ts-sdk/pull/462

Reply:
@hardsetting can you give guidance here?  `variant` is private on AnySignature and AnyPublicKey

Reply:
We could make the `variant` into public readonly.

What is the use case for `verifySecp256k1Account`?
Depending on that, using `instanceof` should be fine.
For example, if it's expected to be used for accounts instantiated or deserialized by the same application (which is probably be the case), we should be good.

Reply:
I worked around this for now